float32 midtop_mid
float32 all_force
float32 bot_Med
float32 bot_lat
bool stance_flg